homeassistant/components/lcn/cover.py

```diff
@@ -122,37 +137,47 @@ def __init__(self, config, address_connection):
         self.motor_port_onoff = self.motor.value * 2
         self.motor_port_updown = self.motor_port_onoff + 1
 
-        self._closed = None
+        self._current_cover_position = 50
 
     async def async_added_to_hass(self):
         """Run when entity about to be added to hass."""
         await super().async_added_to_hass()
         await self.address_connection.activate_status_request_handler(self.motor)
 
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_STOP
+
     @property
     def is_closed(self):
         """Return if the cover is closed."""
-        return self._closed
+        return self._current_cover_position == 0
+
+    @property
+    def current_cover_position(self):
```

Comment:
If we implement the `assumed_state` property and set it to `True`, does the frontend handle that ok? That would be a more appropriate solution.

https://github.com/home-assistant/core/blob/ad6e21182ecc3f6ae883fe826fa339e5188ae9e1/homeassistant/helpers/entity.py#L212-L215